addons/bus/static/src/js/services/bus_service.js

```diff
@@ -80,7 +80,12 @@ var BusService =  CrossTab.extend(ServicesMixin, {
      * @param {function} [callback] if given callback will be called when user clicks on notification
      */
     _sendNativeNotification: function (title, content, callback) {
-        var notification = new Notification(title, {body: content, icon: "/mail/static/src/img/odoobot_transparent.png"});
+        var notification = new Notification(
+            _.unescape(title),
```

Comment:
Indeed, though that's a bit confusing, as our own Notification object does require escaping. Maybe it's worth an extra comment in the code, as a reminder that this is OK **only** because we are using the native Notification API.

How about the `body`, BTW?

Reply:
Body should also be unescaped.

Reply:
The body should be safe because, unlike the title, we use `mailUtils.stripHTML` on the body in `_notifyIncomingMessage`
Unescaping doesn't cost much though...